project_generator/commands/__init__.py

```diff
@@ -11,3 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+import argparse
+from os.path import exists
+
+def argparse_filestring_type(string):
```

Comment:
This one shall be - PEP8 . Taken from there: ``Compound statements (multiple statements on the same line) are generally discouraged.``

Just follow the one that is in this module. We might want to turn pep8 validator and fix some problems. That can be feature request